test.md

```diff
@@ -270,6 +270,38 @@ print it and run it through the `markdownify` Liquid filter. Here's a demo:
 </div>
 </div>
 
+#### Synchronizing multiple tab groups
```

Comment:
We discussed this a bit yesterday. The `data-group` thing is my invention, and not really something that bootstrap has out of the box. If you think this is easier to use let's leave it as is and I can tweak it some other time.